definitions/variable/electricitysources_tags.yaml

```
@@ -0,0 +1,28 @@
+# This file adds needed variables for the EMPIRE data upload
+-  Secondary Energy|Electricity|{Electricity Source By Technology}:
+     description: Net electricity generation from {Electricity Source By Technology}
+     unit: EJ/yr
+- Investment|Energy Supply|Electricity|{Electricity Source By Technology}:
+    description: Investment in {Electricity Source By Technology}
+    unit: billion USD_2010/yr
+- Investment|Energy Supply|Electricity|{Electricity Source}:
+    description: Investment in {Electricity Source}
+    unit: billion USD_2010/yr
+- Variable Cost|Electricity|{Electricity Source By Technology}:
+    description: Variable cost for generating electricity from {Electricity Source By Technology} with the fuel cost
```

Suggest to use a different description than repeating the variable name
```suggestion
    description: Short-term operational cost for generating electricity from {Electricity Source By Technology}
      including fuel cost
```